### ISSUE TITLE
Keikoproj/aws-auth not building

### ISSUE THREAD

**Issue by gabdants** [keikoproj/aws-auth#19]
I'm trying to build the binary but I can't due to this error. I've tryed to add this `context.Context` object to the function, but it still return me errors.

I'm using go 1.15

```
github.com/keikoproj/aws-auth/pkg/mapper/configmaps.go:35:56: not enough arguments in call to k.CoreV1().ConfigMaps(AwsAuthNamespace).Get
	have (string, "k8s.io/apimachinery/pkg/apis/meta/v1".GetOptions)
	want (context.Context, string, "k8s.io/apimachinery/pkg/apis/meta/v1".GetOptions)
github.com/keikoproj/aws-auth/pkg/mapper/configmaps.go:67:63: not enough arguments in call to k.CoreV1().ConfigMaps("kube-system").Create
	have (*"k8s.io/api/core/v1".ConfigMap)
	want (context.Context, *"k8s.io/api/core/v1".ConfigMap, "k8s.io/apimachinery/pkg/apis/meta/v1".CreateOptions)
github.com/keikoproj/aws-auth/pkg/mapper/configmaps.go:92:58: not enough arguments in call to k.CoreV1().ConfigMaps(AwsAuthNamespace).Update
	have (*"k8s.io/api/core/v1".ConfigMap)
	want (context.Context, *"k8s.io/api/core/v1".ConfigMap, "k8s.io/apimachinery/pkg/apis/meta/v1".UpdateOptions)

```

**Comment by eytan-avisror** [comment]
@gabdants are you trying to build master branch?
Are you using `make build` command? it works fine for me.

```
$ make build
CGO_ENABLED=0 go build -ldflags "-X github.com/keikoproj/aws-auth/cmd/cli.buildDate=`date +%FT%T%z` -X github.com/keikoproj/aws-auth/cmd/cli.gitCommit=`git rev-parse HEAD`" -o bin/aws-auth github.com/keikoproj/aws-auth
go: downloading k8s.io/client-go v0.16.9
go: downloading golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
go: extracting golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
go: extracting k8s.io/client-go v0.16.9
go: downloading golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
go: downloading github.com/imdario/mergo v0.3.7
go: extracting github.com/imdario/mergo v0.3.7
go: extracting golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
go: downloading golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a
go: extracting golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a
go: finding k8s.io/api v0.16.9
go: finding k8s.io/apimachinery v0.16.9
go: finding golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
go: finding k8s.io/client-go v0.16.9
go: finding golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
go: finding golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a
go: finding github.com/imdario/mergo v0.3.7
chmod +x bin/aws-auth
```

**Comment by eytan-avisror** [comment]
fixed via #21

**Comment by gabdants** [comment]
thanks @eytan-avisror !
